wordpress-mobile/gutenberg-mobile
# Remove left border of the toolbar

Targets gutenberg PR https://github.com/WordPress/gutenberg/pull/20830

To test:
- Checkout this branch
- Boot the app on android and iOS
- Check that the left border of the bottom toolbar is not visible anymore
- Check that the block toolbar (visible when you edit a block with richtext) is displayed properly

![image](https://user-images.githubusercontent.com/230230/76526311-9effd280-646d-11ea-84c0-34d01879f007.png)


PR submission checklist:
